### PR TITLE
fix(geodb): load at startup and log configuration used

### PR DESCRIPTION
**Pull request by jrgm**
Commit and comments say it all.

r? - @vladikoff

### CODE REVIEW

File: lib/geodb.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ var ACCURACY_MIN_KM = 25
 * `location` data. On failure, returns an empty object
 **/
 module.exports = function (log) {
+
+  log.info({ op: 'geodb.start', enabled: config.enabled, dbPath: config.dbPath })
+
   return function (ip) {
```
Comment (jrgm) [inline lib/geodb.js]: Not part of this PR, but maybe this should be a singleton, fwiw.

File: bin/key_server.js [hunk 2]
```diff
@@ -5,11 +5,23 @@
...
+var getGeoData = require('../lib/geodb')(log)
+
 function main() {
-  var log = require('../lib/log')(config.log.level)
   var metricsContext = require('../lib/metrics/context')(log, config)
   log.setMetricsContext(metricsContext)
 
+  // Force the geo to load and run at startup, not waiting for it to run on
+  // some route later.
+  if (config.env === 'prod' || config.env === 'stage') {
```
Comment (vladikoff) [inline bin/key_server.js]: @jrgm should we just do this for all envs? 

* to avoid more branches between dev and prod
* it will fail safely if no db, etc

Comment (jrgm) [inline bin/key_server.js]: Changed and repushed.

**Comment by vladikoff** [comment]
👍
